ssdutil/main.py

```diff
@@ -6,20 +6,49 @@
 #
 
 try:
-    import argparse
     import os
     import sys
+    import argparse
+    import json
+    import psutil
+    from blkinfo import BlkDiskInfo
 
-    from sonic_py_common import device_info, logger
+    from sonic_py_common import device_info, syslogger
 except ImportError as e:
     raise ImportError("%s - required module not found" % str(e))
 
 DEFAULT_DEVICE="/dev/sda"
 SYSLOG_IDENTIFIER = "ssdutil"
+DISK_TYPE_SSD = "sata"
 
 # Global logger instance
-log = logger.Logger(SYSLOG_IDENTIFIER)
+log = syslogger.SysLogger(SYSLOG_IDENTIFIER)
+
+def get_default_disk():
+    """Check default disk"""
+    default_device = DEFAULT_DEVICE
+    host_mnt = '/host'
+    host_partition = None
+    partitions = psutil.disk_partitions()
```

Comment:
It might good to check, if 'partitions' is valid before proceeding?

Reply:
Fixed in latest, thanks for the catch :)